### PR TITLE
Engagement/jessica/14613 indiana migration pre work

### PR DESCRIPTION
**Pull request by JessicaWNava**
This PR implements the pre migration work for Indiana and updates the Simple Report sender transform, resulting in changes to the integration tests as well. 

Test Steps:
1. Review that the changes in organizations.yml match IN previous settings
2. Check that the resulting message matches the outbound message from the Covid pipeline
3. Make sure the tests pass

## Changes
- Added settings for Montana to organizations.yml
- Created the IN receiver transform
- Updated the simple report sender transform
- Updated documentation for the overall pre migration process
- Updated integration tests to reflect the changes in the sender transform

### Testing
- [x] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ ] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [x] Added tests?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should test/verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Linked Issues
- Fixes #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?

### CODE REVIEW

File: prime-router/settings/STLTs/IN/in-phd.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
+      organizationName: "in-phd"
+      topic: "full-elr"
+      customerStatus: "testing"
+      translation:
+        schemaName: "classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml"
```
Comment (oslynn) [inline prime-router/settings/STLTs/IN/in-phd.yml:16]: Do you have prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/IN/IN-receiver-transform.yml schema for this?

File: prime-router/settings/STLTs/IN/in-phd.yml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        numberPerDay: 1440
+        initialTime: "01:20"
+        timeZone: "EASTERN"
+        maxReportCount: 100
+        whenEmpty:
+          action: "NONE"
+          onlyOncePerDay: false
+      description: ""
+      transport:
+        host: "sftp"
```
Comment (oslynn) [inline prime-router/settings/STLTs/IN/in-phd.yml:92]: Do you have the real endpoint for this?

File: prime-router/settings/STLTs/MT/mt-doh.yml [hunk 3]
```diff
@@ -0,0 +1,98 @@
+- name: "mt-doh"
+  description: "Montana Department of Health"
```
Comment (oslynn) [inline prime-router/settings/STLTs/MT/mt-doh.yml:2]: Would you please make MT into it's own PR?

Comment (JessicaWNava) [reply prime-router/settings/STLTs/MT/mt-doh.yml:2]: It is its own PR. I think these are stragglers. Will remove.

**Review by oslynn — APPROVED** [review]
Looking very good to me.

**Review by GilmoreA6 — APPROVED** [review]

**Review by oslynn — APPROVED** [review]

**Comment by JessicaWNava** [comment]
@GilmoreA6 It adds 19.1.1 which is the date/time of the analysis

**Review by GilmoreA6 — APPROVED** [review]
